keyboards/planck/keymaps/gunp/upload-and-make.sh

```diff
@@ -2,15 +2,15 @@
 
 pushd ~/git-repos/qmk_firmware/
 rm -rf keyboards/planck/keymaps/gunp/
-cp -rf ~/Documents/qmk-planck/ keyboards/planck/keymaps/gunp/
-if [[ $1 == "dfu" ]]; then
+cp -rf ~/git-repos/gunp-geek-repo/qmk-planck/ keyboards/planck/keymaps/gunp/
+if [[ $2 == "dfu" ]]; then
     # sudo dfu-programmer atmega32u4 erase --force
     # sudo dfu-programmer atmega32u4 flash .build/planck_rev4_gunp.hex
     # sudo dfu-programmer atmega32u4 reset
-    sudo make planck/rev4:gunp:dfu
+    sudo make planck/$1:gunp:dfu
```

Comment:
Sudo shouldn't be used.   It's a bad practice. 

Instead, you should do this:
https://docs.qmk.fm/#/faq_build?id=linux-udev-rules